src/gateway/request.js

```diff
@@ -40,10 +47,15 @@ export const abortInFlightRequests = (): void => {
  * have the abort function. Therefore, you'll need to readd it.
  */
 export const request = (
-    options: RequestOptions,
     logger: Logger,
     url: string,
+    options?: RequestOptions,
```

Comment:
They are optional now, so they have to go at the end. This is more idiomatic, I think, anyway.